theme/static/style/style.css

```diff
@@ -498,6 +608,11 @@ pre code {
         are different than the tablet section! No duplication!
     */
 
+    /*navigation section*/
+    .topnav img {
+        width: 200%;
```

Comment:
This will stretch the image but the image itself isn't big enough. I think we need a different approach here.

Reply:
Remove this block since after the updated this selector will not select anything.